docs/advanced/writing-declarations.md

```diff
@@ -530,6 +532,27 @@ const v3 = (v1 * 4) as Vector;
 const d = v3.dot(v2);
 ```
 
+There are also language extensions that allow declaring special functions which will transpile to operators. For example:
```

Comment:
Seems like duplicate information, couldn't you just refer to the language extensions page? `./language-extensions.md#operator-map-types` should do the trick